server/ai_mediaserver.go

```diff
@@ -859,6 +860,9 @@ func (ls *LivepeerServer) UpdateLiveVideo() http.Handler {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		}
+
+		w.Header().Set("Access-Control-Allow-Origin", "*")
+		w.WriteHeader(http.StatusOK)
```

Comment:
Curious why explicitly setting this here is needed since I think StatusOK is the default.

Reply:
I re-tested it and confirmed it works with a 200 response by default. It must have been something else mixed in causing issues before.

<img width="384" height="305" alt="image" src="https://github.com/user-attachments/assets/1630b073-bef5-4ad6-b6a5-463704873f1e" />
<img width="416" height="254" alt="image" src="https://github.com/user-attachments/assets/05e60876-9763-4bb3-8e0d-05b884e194ff" />